templates/docs/base/forms.md

```diff
@@ -124,7 +124,10 @@ The `<input type="range">` allows a user to select from a specified range of val
 View example of the slider pattern
 </a></div>
 
-## Fieldset
+<h2>
+  Fieldset
+  <div class="p-status-label--positive u-align--middle">New</div>
+</h2>
```

Comment:
I don't think fieldset is new, isn't it just recently updated to have "required" styling in #5214 ? Maybe it is "Updated" then?